src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -752,9 +760,31 @@ <h4 class="p2">Products that have been bought together</h4>
       </template>
     </amp-list>
     <!--
-    The button `tap` action sets the value of the `relatedProductsSrc` to an extended list of related products.
+    While infinite scrolling is not possible at the moment in AMP, you can use `amp-bind` to dynamically change the
+    src of `amp-list` and add more items to the page.
+    We bind the `src` attribute to the id of an `amp-state` component so that the `amp-list` will use the items
+    from that component as a `src`. We append items to the `amp-state` everytime the user click on the Show more button, see below for more info.
```

Comment:
Can you add a sentence describing the trick you've been using to init the state with the same content as the amp-list.